views/content-cross-linked.phtml

```diff
@@ -1,3 +1,16 @@
 <?= $data->content ?>
 <hr>
-<p><em>Also published on <a href="<?= $data->medium_post->url ?>">Medium</a>.</em></p>
+<p>
+	<em>
+		<?php
+		printf(
```

Comment:
Yah, I do realise that, and I'm not saying it just for the sake of it. I may not have written much PHP recently, but I've done conservatively well over 100k lines of it in my career, and old habits die hard. Seeing as I'm going to be the primary maintainer, I'd rather stick with the convention I've developed, even though that does contradict the WP standards. (My one concession was underscored_variables, because i have to interact with so many WP functions and vars. $camelCase for life.)